src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -417,6 +417,65 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         execCond("docker rmi -f ${id}") || true
     }
 
+    def 'outputs from default distribution locations are added to the docker context'() {
```

Comment:
Johannes, I am trying to understand your problem. Here is a test that suggests what I have been proposing from the start of this discussion: it declares a dependency on a standard Gradle `distTar` task and demonstrates that one can build a Docker image from the resulting tar file: https://github.com/palantir/gradle-docker/compare/rfink/verify-standard-gradle-distribution-plugin?expand=1

The Docker image built here contains the contents of the tar file as expected per the Dockerfile:
```
can-build-Docker-image-from-standard-Gradle-distribution-plugin_6
/tmp # cd can-build-Docker-image-from-standard-Gradle-distribution-plugin_6/
/tmp/can-build-Docker-image-from-standard-Gradle-distribution-plugin_6 # ls
bin  lib
/tmp/can-build-Docker-image-from-standard-Gradle-distribution-plugin_6 # find .
.
./lib
./lib/can-build-Docker-image-from-standard-Gradle-distribution-plugin_6.jar
./bin
./bin/can-build-Docker-image-from-standard-Gradle-distribution-plugin_6
./bin/can-build-Docker-image-from-standard-Gradle-distribution-plugin_6.bat
/tmp/can-build-Docker-image-from-standard-Gradle-distribution-plugin_6 #
```

Please let me know if this clarifies the usage. I agree that the plugin code (including tests) have grown a bit out of control over the past months and we're happy to fix bugs and make improvements where needed. As such, we have indeed had good luck with the plugin and have been using it happily.